### PR TITLE
Better list

### LINKED ISSUE #16: Feature/Rework: Use discord embeds when listing out music
https://tutorial.vcokltfre.dev/tutorial/10-embeds/

There are better ways to list out the music on the queue; as of now, we use prettytables (which do work, but when you resize the discord window, or if the window is small/narrow, it doesn't look all that great).

### PR DESCRIPTION
**Pull request by duckduckdoof**
Smaller issue, I was using prettytables to print out the queue, and the current playing song; but, I looked at discord EMBEDS and was like wow that's so much better.

### CODE REVIEW

File: discord_yt_audio_bot.py [hunk 1]
```diff
@@ -168,26 +172,38 @@ async def queue( ctx, url ):
...
-        music_list = list( yt_queue.queue )
-        for i, music in enumerate(music_list):
-            queue_table.add_row( [ music.title ] )
-        table_str = '`' + queue_table.get_string() + '`'
-
-    # Print out the strings
-    await ctx.send( current_str + table_str )
+        yt_queue_list = list( yt_queue.queue )
+        queue_val = ""
+        for i in range( 0, len(yt_queue_list) - 1 ):
```
Comment (Lmenden) [inline discord_yt_audio_bot.py]: The power of `join` compels you!

Comment (Lmenden) [reply discord_yt_audio_bot.py]: (To rewrite this section using `join`)

Comment (duckduckdoof) [reply discord_yt_audio_bot.py]: I feel compelled to JOIN the `join`

Comment (Lmenden) [reply discord_yt_audio_bot.py]:
```
>>> a = ['song a', 'song b', 'song c']
>>> b = [f'{i+1}:\t {song}' for i, song in enumerate(a)]
>>> print('\n'.join(b))
1:       song a
2:       song b
3:       song c
>>>
```

(After this is updated, I approve the PR)

**Comment by Lmenden** [comment]
Approved!

**Pull request merged**